web/src/components/Plugin/UI/cors.tsx

```diff
@@ -40,8 +41,12 @@ export const FORM_ITEM_WITHOUT_LABEL = {
   },
 };
 
-const Cors: React.FC<Props> = ({ form }) => {
+const Cors: React.FC<Props> = ({ form, schema }) => {
   const { formatMessage } = useIntl();
+  const properties = schema?.properties
```

Comment:
If we don't pass `schema`, `properties` will be undefined 🤔 right?

Should we have a check that this Property must be passed